lispyville-test.el

```diff
@@ -14,6 +15,9 @@
 
 (setq evil-move-cursor-back nil)
 
+;; FIXME setting `evil-want-C-u-delete' directly does not take effect
```

Comment:
You could `setq` it before the `(require 'evil)`, and it would work, but I'd rather it only apply to this test. Add a `customize-set-variable` to set it in the beginning of the `ert-deftest`, and then set it back to nil afterwards.